SnooksHarvard.csl

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
```

Comment:
we'll want this with a default locale, I think. en-GB would make most sense given the similarities (and the fact that we don't have--nor seem to need--an Australia-specific locale.
So add default-locale="en-GB" to this line.